theme/css/chrome.css

```diff
@@ -500,3 +500,7 @@ ul#searchresults span.teaser em {
     border-top-left-radius: inherit;
     border-top-right-radius: inherit;
 }
+
+.nav-return:hover {
+    color: var(--sidebar-active);
+}
```

Comment:
@aramikm   just need a new line here lint purpose

Reply:
Applied. when I run `npm run lint` it does not show up in errors.
How do you check linting?

Reply:
Yes current lint does not work on static files like css/html we can add a issue for later but your PR looks good, feel free to merge it